### DISCUSSION TITLE
r/maxus-the-guide-to-rv-electricity-usage

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#52] [Announcements]
# r/maxus-the-guide-to-rv-electricity-usage

房车用电指南，两年实际房车使用的真实体验

https://blog.ops-coffee.cn/r/maxus-the-guide-to-rv-electricity-usage

**Comment by tjsky** [comment]
家里有亲戚跑大车的，之前和他聊起来，感觉长途卡车在用电上感觉和房车很像，和你的房车的区别只是他是个厢式车有足够面积，装了接近700W的太阳能板。

**Reply by ops-coffee** [reply]
700瓦那应该很管用了，天气好的情况下一天能发3.5度电左右，差不多够一天用了

**Comment by Alexanderklau** [comment]
哥，我真羡慕你啊！！！你就是我努力的目标了，我现在一周wfh 3天，未来我一定要跳槽到能永久wfh的地方！哥就是我努力的目标！

**Reply by ops-coffee** [reply]
我真是羡慕你，一周能wfh 3天，而我要天天上班，不过可能离不上班也不远了，裁员Warning
